tests/test_utils_python.py

```diff
@@ -104,6 +104,10 @@ def test_one_with_encoding(self):
     def test_real_binary_bytes(self):
         assert not binary_is_text(b"\x02\xa3")
 
+    def test_string_contain_nonprintable(self):
```

Comment:
The name does not fit the purpose of the test. As of how I understand, only the characters with binary value 0 to 32 (except '\0', '\t', '\n', '\r') are considered unprintable. What this test actually checks is that a TypeError is raised when the input has type str instead of bytes.

Reply:
i agree with maja on this, both the name and what this test against should be clear

Reply:
You are correct, I was confused here. Thanks!